### PR TITLE
Release/6.10.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
### Added

- Internal: Added note about lockfileVersion and npm version requirement for SDK contributors in CONTRIBUTING.md documentation.
- Public: Added support for Italian `it_IT` and Portuguese `pt_PT`.
- Internal: Added support for visual regression testing using Percy.
- Internal: Resolved inaccurate reporting on test automation failures.
- Internal: Added ability to execute/ignore tests via the use of tags/regex.

### Changed

- UI: Update Face Liveness Video Challenge screen UI
- UI: Updated default Welcome screen UI
- Public: Move documentation for SDK UI Customizations and Premium Enterprise Features into separate Markdown files to reduce README size
- Public: Updated SDK copy for English (en_US), Spanish (es_ES), French (fr_FR) and German (de_DE). For details on what keys/copy have changed please refer to the MIGRATION.md documentation.
- Internal: Upgrade sass (Dart Sass) from `1.26.9` to `1.33.0`
- Internal: Upgrade stylelint from `13.6.1` to `13.13.1` as well as stylelint-config-sass-guidelines from `7.0.0` to `8.0.0` and stylelint-scss from `3.18.0` to `3.19.0`

### Fixed

- Internal: Upgrade Preact from version `10.5.4` to `10.5.13` in order to resolve an unhandled exception on reinitialising the SDK after closing the SDK modal for some integrations when using Modal mode.
- Public: Fix Country Selection screen not displaying when SDK is initialised with boolean `documentTypes` configuration.
- Public: Fixed issue where `onComplete` callback was fired without the necessary data for the `face` step. Added exception to `onError` callback to inform that `onComplete` could not be fired due to missing data.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +57,10 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Added user consent content reload screen
 - Public: When `photoCaptureFallback` option is disabled in the face step, the requested variant is video and browser does not support MediaRecorder, attempt to redirect to the cross-device flow or display the unsupported browser error.
 - Internal: Refactor for better reusability of video capture components.
+- Internal: Added support for testing across multiple browsers.
```
Comment (kopijunkie) [inline CHANGELOG.md:60]: This line seems to be duplicated in line 33 under v6.9.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +57,10 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
...
+- Internal: Added support for visual regression testing using Percy.
+- Internal: Resolved inaccurate reporting on test automation failures.
+- Internal: Added ability to execute/ignore tests via the use of tags/regex.
```
Comment (kopijunkie) [inline CHANGELOG.md:63]: These feel like they should be under current release. @SherBhachuOnFido these are unreleased changes still, right?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -110,19 +111,20 @@ Some of the SDK configuration options can also be previewed in the demo app by u
 | `useHistory`               | `true`,`false`      | `false` | Enable basic dummy host app history                              |
 | `customisedUI`             | `true`,`false`      | `false` | Enable customised SDK UI                                         |
 | `showUserConsent`          | `true`,`false`      | `false` | Enable user consent step                                         |
+| `customWelcomeScreenCopy`  | `true`,`false`      | `false` | Configure welcome screen with custom text                        |
```
Comment (kopijunkie) [inline CONTRIBUTING.md:114]: Missing documentation for new `multiDocWithBooleanValues` URL query parameter, think my commit for the change must have been after PR was merged.
```suggestion
| `customWelcomeScreenCopy`  | `true`,`false`      | `false` | Configure welcome screen with custom text                        |
| `multiDocWithBooleanValues`   | `true`,`false`      | `false` | Enable document types with boolean flags                     |
```

File: MIGRATION.md [hunk 4]
```diff
@@ -82,6 +152,8 @@ The **English** copy for the following string(s) has been added:
 
 ### Added strings
 
+The **English**, **Spanish**, **German**, and **French** copy for the following string(s) has changed:
+
```
Comment (kopijunkie) [inline MIGRATION.md:156]: Stray duplicated line(s)? 🤔

**Review by kopijunkie — CHANGES_REQUESTED** [review]
Mostly just documentation fixes. Looks like some stray/duplicated release notes & unreleased features ending up in previous releases, most likely from merges.

**Review by kopijunkie — APPROVED** [review]

**Review by ethan605 — APPROVED** [review]

**Pull request merged**